ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS 2.xx.x branch released xxxx-xx-xx
+
+Bugfix
+   * Fix compilation issue on ARM platform before armv6. The `umaal` arm 
+     instruction is only available on ARM cpu with DSP starting with armv6.
```

Comment:
Please credit yourself in the ChangeLog entry
```suggestion
      instruction is only available on ARM cpu with DSP starting with armv6. Fixed by vhenriet #2324.
```

Reply:
Unless you prefer to use your real name

Reply:
@vhenriet Do you have time to adapt the ChangeLog as my colleague @RonEld suggested?